lib/distributions.sh

```diff
@@ -549,7 +549,10 @@ install_distribution_specific()
 
 		;;
 	bionic|eoan|focal)
-
+	
+			#add ppa repository
+			if [[ ${DDE_DESKTOP} -eq "yes" ]] && add-apt-repository ppa:ubuntudde-dev/stable
```

Comment:
just kidding it should be:
`[[ ${DDE_DESKTOP} -eq "yes" ]] && add-apt-repository ppa:ubuntudde-dev/stable`

just remove the if